newspack-listings.php

```diff
@@ -32,6 +32,11 @@
 require_once NEWSPACK_LISTINGS_PLUGIN_FILE . '/includes/class-newspack-listings-taxonomies.php';
 require_once NEWSPACK_LISTINGS_PLUGIN_FILE . '/includes/class-newspack-listings-api.php';
 
+// Enable experimental/in-progress self-serve listings functionality.
+if ( defined( 'NEWSPACK_LISTINGS_DEV_MODE' ) && NEWSPACK_LISTINGS_DEV_MODE ) {
```

Comment:
Suggestion: if there would be more feature-flagged functionalities, the flag could be more specific, like `NEWSPACK_LISTINGS_SELF_SERVE_PRODUCTS`

Reply:
Good idea, updated in 9da73ca.